README.md

```diff
@@ -1,50 +1,49 @@
 # API Example
 
-Example app using Climate's API.
+Example apps using Climate's API.
 
 ## Setup
 
-Install python 3.6+.
+1. Install python 3.6+.
 
-```bash
-# if you use Mac OS X and brew, this can be done with:
-brew install python3
-```
-
-Make a virtual environment, activate it, and install dependencies.
-
-```bash
-python3 -m venv api-example
-source api-example/bin/activate
-pip install -r requirements.txt
-```
-
-When you're done testing the example, deactivate the virtual environment with:
-
-```bash
-deactivate
-```
+    ```bash
+    # if you use Mac OS X and brew, this can be done with:
+    brew install python3
+    ```
 
-## Running the example
+2. Make a virtual environment, activate it, and install dependencies.
 
-1. Follow steps in _Setup_
+    ```bash
+    python3 -m venv api-example
+    source api-example/bin/activate
+    pip install -r requirements.txt
+    ```
 
-1. Set the following environment variables to the values provided by Climate:
+3. Set the following environment variables to the values provided by Climate:
 
     ```
     export CLIMATE_API_ID="partner-fooco"
     export CLIMATE_API_SECRET="azbq56fpadhnt8oukoeani2a4w"
     export CLIMATE_API_KEY="partner-fooco-216b9875-0158-4142-1ab2-7c3bdbd6a2157"
     ```
+When you're done testing the example, deactivate the virtual environment with:
+
+    ```bash
```

Comment:
this is formatting oddly in markdown. i think you'll want to remove the 4 space indentation.